File: app/src/main/java/com/asmat/rolando/popularmovies/adapters/FavoriteMoviesGridAdapter.java
package com.asmat.rolando.popularmovies.adapters;

import com.asmat.rolando.popularmovies.R;
import com.asmat.rolando.popularmovies.models.MovieAdapterOnClickHandler;

public class FavoriteMoviesGridAdapter extends MoviesGridBaseAdapter {
    public FavoriteMoviesGridAdapter(MovieAdapterOnClickHandler handler) {
        super(handler);
    }

    @Override
    public int getEmptyStateLayoutID() {
        return R.layout.empty_state_favorites;
    }
}

File: app/src/main/java/com/asmat/rolando/popularmovies/ui/adapters/grid/FavoriteMoviesGridAdapter.kt
package com.asmat.rolando.popularmovies.ui.adapters.grid

import com.asmat.rolando.popularmovies.R
import com.asmat.rolando.popularmovies.ui.adapters.MovieAdapterOnClickHandler

class FavoriteMoviesGridAdapter(handler: MovieAdapterOnClickHandler) : BaseMoviesGridAdapter(handler) {

    override val emptyStateLayoutID: Int
        get() = R.layout.empty_state_favorites

}